src/components/2_molecules/navigation/primaryNav.js

```diff
@@ -59,137 +80,103 @@ class MainSiteNav extends React.Component {
-                    node.switch_on ? "dropdown" : ""
-                  } ${_.kebabCase(node.title)}`}
-                  role="menuitem"
-                  data-submenu={
-                    node.switch_on ? _.kebabCase(node.switch_on) : undefined
-                  }
-                  onClick={
-                    node.title === "Mobile"
-                      ? this.openMobileNav
-                      : this.openSubMenu
-                  }
-                  tabIndex={0}
-                >
-                  {node.title === "Search" ? (
-                    <span className="header-search-icon">
-                      <span className="visually-hidden">{node.title}</span>
+      <div className="o-layout__col">
+        <div className="o-layout__module c-main-menu">
+          <nav
+            role="menu"
+            className="o-menu o-menu--flex"
+            aria-expanded="false"
+          >
+            <div className="menu-header-primary-container">
+              <ul id="menu-header-primary" className="o-menu__list">
+                <li className="o-menu__item o-menu__item--why-linode" data-submenu="sub-menu--why-linode">
+                  <a className="o-menu__link" href="#sub-menu--why-linode" target="_self" data-submenu="sub-menu--why-linode">
+                    <span className="o-menu__title" data-submenu="sub-menu--why-linode">
+                      <span className="title" data-submenu="sub-menu--why-linode">
+                        Why Linode
+                      </span>
+                    </span>
+                  </a>
+                </li>
+                <li className="o-menu__item o-menu__item--products" data-submenu="sub-menu--products">
+                  <a className="o-menu__link" href="#sub-menu--products" data-submenu="sub-menu--products">
+                    <span className="o-menu__title" data-submenu="sub-menu--products">
+                      <span className="title" data-submenu="sub-menu--products">
+                        Products
+                      </span>
+                    </span>
+                  </a>
+                </li>
+                <li className="o-menu__item o-menu__item--solutions">
+                  <a className="o-menu__link" href="https://linode.com/solutions">
+                    <span className="o-menu__title">
+                      <span className="title">
+                        Solutions
+                      </span>
+                    </span>
+                  </a>
+                </li>
+                <li className="o-menu__item o-menu__item--marketplace">
+                  <a className="o-menu__link" href="https://linode.com/marketplace">
+                    <span className="o-menu__title">
+                      <span className="title">
+                        Marketplace
+                      </span>
+                    </span>
+                  </a>
+                </li>
+                <li className="o-menu__item o-menu__item--pricing">
```

Comment:
There's a lot of repetition here. You can iterate through a list instead and adjust the classes. But also: does each section have a different style? Having a separate class for each li is a little nonstandard.